tests/e2etest-xnetwork/main.k

```diff
@@ -0,0 +1,24 @@
+
+import models.io.upbound.dev.meta.v1alpha1 as metav1alpha1
+import models.io.upbound.gcp.compute.v1beta1 as computev1beta1
+import models.io.upbound.gcp.compute.v1beta2 as computev1beta2
+import models.io.upbound.gcp.v1alpha1 as gcpv1alpha1
+import models.io.upbound.gcp.v1beta1 as gcpv1beta1
+import models.io.upbound.platform.gcp.v1alpha1 as platformgcpv1alpha1
+import models.k8s.apimachinery.pkg.apis.meta.v1 as metav1
+
+
+_items = [
```

Comment:
how we setup the proidc trusts in gcp for e2e Tests?

So we need something Like as pre-req:

```
gcloud iam workload-identity-pools create upbound-oidc-pool \
  --location="global" \
  --display-name="upbound-oidc-pool" \
  --description="An identity provider for Upbound" \
  --disabled=False
```

```
gcloud iam workload-identity-pools providers create-oidc upbound-oidc-provider \
  --location="global" \
  --workload-identity-pool="upbound-oidc-pool" \
  --display-name="upbound-oidc-provider" \
  --description="OIDC provider for Upbound" \
  --issuer-uri="https://proidc.upbound.io" \
  --attribute-mapping="google.subject=assertion.sub" \
  --attribute-condition='google.subject.contains("mcp:")' \
  --allowed-audiences="sts.googleapis.com"
```

Reply:
removed e2e for now